data/en/2021/portuguese/agenda.yml

```diff
@@ -11,4 +11,42 @@ types:
 items:
     - name: Keynote
       presenter: Program Committee
-      type: Keynote
+      type: keynote
+      time: 09 AM BRT / 1 PM (WEST)
```

Comment:
A tiny thing, I assume you might miss a parenthesis for BRT and a slash between the 2 times? All other values in time are this kind of format: 10 AM (BRT) // 2 PM (WEST)